client/rest/src/plugins/rosetta/accountRoutes.js

```diff
@@ -52,43 +81,61 @@ export default {
 
 			const startChainHeight = await getChainHeight();
 
-			const mosaics = await services.proxy.fetch(`accounts/${address}`, json => json.account.mosaics);
+			const promises = [services.proxy.fetch(`accounts/${address}`, json => json.account.mosaics)];
+			if (typedRequest.include_mempool)
+				promises.push(services.proxy.fetchAll('transactions/unconfirmed?embedded=true', PAGE_SIZE));
+
+			const [mosaics, unconfirmedTransactions] = await Promise.all(promises);
 
 			const endChainHeight = await getChainHeight();
 
 			if (startChainHeight !== endChainHeight)
 				throw RosettaErrorFactory.SYNC_DURING_OPERATION;
 
+			let amounts = await mapMosaicsToRosettaAmounts(mosaics);
+			if (typedRequest.include_mempool && unconfirmedTransactions?.length) {
```

Comment:
i don't think you need ? because you should always have unconfirmedTransactions when include_mempool is true